### ISSUE TITLE
Unable to install with composer on Lumen 5.5

### ISSUE THREAD

**Issue by retnek** [websightgmbh/l5-google-cloud-storage#17]
When I call `composer require websight/l5-google-cloud-storage` I got the following output:

> Your requirements could not be resolved to an installable set of packages.
>   Problem 1
>     - Installation request for websight/l5-google-cloud-storage ^1.1 -> satisfiable by websight/l5-google-cloud-storage[v1.1.0].
>     - Conclusion: remove illuminate/contracts v5.5.28
>     - Conclusion: don't install illuminate/contracts v5.5.28
> ...

It is not compatible with Lumen 5.5.2?

**Comment by cedricziel** [comment]
Master is compatible. Yet this package switched to another flysystem driver and the implementation was not completed yet. 

This very much means: Yes it's compatible, but may need some work and your constraint should be `dev-master` instead of `^1.1`.

If you feel you want to use a stable version, I'd also happily review your pull requests :)

**Comment by retnek** [comment]
Yes, You're right! If You use Lumen You need more complex tasks to work with this package, such as define filesystem singleton and Storage class alias in bootstrap/app.php. I'm trying to put it together.

**Comment by cedricziel** [comment]
This package is very simple - if you want to, you can also initialize the flysystem adapter manually

**Comment by retnek** [comment]
Lumen does not include the filesystem (among other typical Laravel services) out of the box. So it must be initialize manually before use this package (as described [here](https://nicksilvestro.net/2016/05/28/adding-laravels-storage-facade-into-lumen/)). But now it seems to work fine with the dev-master.
